src/index.html

```diff
@@ -1,13 +1,65 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html class="page" lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="
+        width=device-width,
+        user-scalable=no,
+        initial-scale=1.0,
+        maximum-scale=1.0,
+        minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body class="page__content">
+    <div class="product-card" data-qa="card">
+      <img
+        class="product-card__image"
+        src="images/imac.jpeg"
+        alt="iMac"
+      >
+
+      <div class="product-card__name">
+        Моноблок APPLE A1419 iMac 27&quot; Retina 5K (MNED2UA/A)
+      </div>
+
+      <div class="product-card__id">Код товара: 195434</div>
```

Comment:
div development ;) next time try to use semantic tags as well, such as h2, section, p, etc